### PR TITLE
Add desk sprites

### PR DESCRIPTION
**Pull request by jutsuu**
Adds a base module for office sprites, with the addition of sprites for desks. 
There's plenty of room for improvement in terms of animation and the overall organization of the sprites modules, that said I'd rather tackle it in a PR dedicated to it. 
 
 Also renames the `world` sprite into `floor`. 
 
image

### CODE REVIEW

**Review by facumendizabal — APPROVED** [review]
Nice desks, LGTM!

File: src/ecs/sprites/draw.rs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+use super::super::systems::{physics_system::physics::*, render_system::camera::Camera};
+use ggez::graphics::spritebatch::SpriteBatch;
+
+pub trait Draw {
```
Comment (mcass19) [inline src/ecs/sprites/draw.rs:4]: ❤️

File: src/ecs/game_state.rs [hunk 2]
```diff
@@ -56,12 +68,56 @@ impl ggez::event::EventHandler<GameError> for GameState {
...
+                            desk
+                        )?;
+                    }
+                    _ => (),
+                },
+                None => (),
+            }
+        }
+
+        draw_interactions(
```
Comment (mcass19) [inline src/ecs/game_state.rs:114]: I think we are drawing the interactions twice

File: src/ecs/systems/render_system/render_system.rs [hunk 3]
```diff
@@ -50,48 +60,35 @@ pub fn draw_world_bounds(ctx: &mut Context, camera: &Camera, world_size: &Size)
...
+
     graphics::draw(ctx, npcs_sprite_batch, draw_param)?;
     npcs_sprite_batch.clear();
-    
+
     Ok(())
 }
 
-pub fn draw_objects(ctx: &mut Context, camera: &Camera, physics_components: &Vec<Option<Physics>>) -> GameResult {
+pub fn draw_objects(
```
Comment (mcass19) [inline src/ecs/systems/render_system/render_system.rs:87]: We're not using this one and the above anymore right?

**Review by mcass19 — COMMENTED** [review]
Awesome!!! Getting closer to have an office at some time 🚀
Left some minor things

**Pull request merged**